README.md

```diff
@@ -280,6 +280,22 @@ To get in touch with the developers,
 or [create an issue](https://github.com/trixi-framework/Trixi.jl/issues/new).
 
 
+## Participating research groups
+Participating research groups in alphabetical order:
+
+[Applied and Computational Mathematics, RWTH Aachen University :de:](https://www.acom.rwth-aachen.de)
+
+[Applied Mathematics, Linköping University :sweden:](https://liu.se/en/employee/andwi94)
```

Comment:
@JoshuaLampert maintains P4est.jl and other repos, so we should also include his affiliation, I think:

```suggestion
[Applied and Computational Mathematics, RWTH Aachen University :de:](https://www.acom.rwth-aachen.de)

[Applied Mathematics, Department of Mathematics, University of Hamburg :de:](https://www.math.uni-hamburg.de/en/forschung/bereiche/am.html)

[Applied Mathematics, Linköping University :sweden:](https://liu.se/en/employee/andwi94)
```